### ISSUE TITLE
OpenImageIO + Python3 ( and 2.7 )

### ISSUE THREAD

**Issue by stephane-lb** [Homebrew/homebrew-science#4413]
Good evening,

I just installed openimagiio with the option below : 

```
brew reinstall openimageio --with-qt --with-python --with-python3 
```

Then I just opened a terminal, launch python3 and try this line : 

```
import OpenImageIO as oiio
```

And I got this error : 

```
ImportError: No module named 'OpenImageIO'
```

So I think something is wrong, but I did not see where by editing the formula, if anyone can help it would be great !

Thanks,

Stephane-lb

**Comment by ilovezfs** [comment]
Please try again with the new version that was just merged. `brew update`

**Comment by stephane-lb** [comment]
Hi @ilovezfs ,

Thanks for your help, I did that : 
``` 
brew update
brew reinstall openimageio --with-qt --with-python --with-python3 
```

Then in Python3 or Python : 
```
>>> import OpenImageIO as oiio
```

and I got the same error

```
ImportError: No module named 'OpenImageIO'
```

Any other idea ?

Thanks,

Stephane-lb

**Comment by ilovezfs** [comment]
Does it happen if you just use --with-python or just use --with-python3?

**Comment by stephane-lb** [comment]
Hi @ilovezfs 

Ok first of all something strange just happened
```
> $ brew reinstall openimageio --with-qt --with-python                                                                                                                           
==> Reinstalling homebrew/science/openimageio with --with-qt, --with-python3
==> Downloading https://github.com/OpenImageIO/oiio/archive/Release-1.7.7.tar.gz
```

For the reste same error ...

Thanks,

Stephane-lb

**Comment by ilovezfs** [comment]
You have to uninstall first, otherwise it will combine the set of original options used when installing it, and whatever new additional options you pass to reinstall.

**Comment by remia** [comment]
I don't know if it's the same problem, but I had the same kind of issue (module not found) with both python 2.7 and python 3.5. I noticed that the formula install the python module at the wrong path, ie. /usr/local/lib/python2.7/python instead of /usr/local/lib/python2.7/site-packages.

Just moving the .so file at the correct location resolved the issue for me.

**Comment by stephane-lb** [comment]
@remia : Thank you! That's fixed !

**Comment by iMichka** [comment]
Fixed with #5938
